standard/attributes.md

```diff
@@ -982,7 +982,8 @@ Specifies that a non-nullable return value may be null.
 
 > *Example*: Consider the following generic method:
 >
-> <!-- Example: {template:"code-in-class-lib", name:"MaybeNull1Attribute", replaceEllipsis:true, customEllipsisReplacements: ["return default;"], expectedErrors:["CS8627"]} -->
+> <!-- TODO:  This sample no longer generates the error in C# 9. This could be part of the new nullable constraints. Was: , expectedErrors:["CS8627"]-->
```

Comment:
I added this TODO note because this example is invalid in C# 9. The first code that the example says doesn't work works fine.

We'll need a new example for this attribute.